tic_tac_toe/game.py

```diff
@@ -33,7 +45,11 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    for x in board:
+        for y in x:
+            if y == '-':
+                return False
+    return True
 
```

Comment:
I find I often use simple variables like x and y too, but I think names help readability.